bundles/org.openhab.binding.pentair/src/main/resources/OH-INF/thing/intellichem.xml

```diff
@@ -96,34 +96,54 @@
 		<item-type>Number</item-type>
 		<label>Tank Level</label>
 		<description>Tank level (1-7).</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Level</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="calciumHardness">
 		<item-type unitHint="ppm">Number:Dimensionless</item-type>
 		<label>Calcium Hardess</label>
 		<description>Calcium hardness (ppm).</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>AirQuality</tag>
```

Comment:
Definitely not AirQuality - maybe Water for most of these since the readings tell the quality of the Water.